js-miniapp-sdk/src/miniapp.ts

```diff
@@ -17,7 +19,7 @@ interface MiniAppFeatures {
   getUniqueId(): Promise<string>;
 
   /** @returns The Promise of permission result of mini app from injected side. */
-  requestLocationPermission(): Promise<string>;
+  requestLocationPermission(permissionDescription: string): Promise<string>;
```

Comment:
I think this still needs the optional: `permissionDescription?: string`. Or else this would be a breaking change (for Typescript users).

Reply:
Sorry, I forgot about to make it optional. I have updated this part with `permissionDescription?: string`.